tests/utils.go

```diff
@@ -1573,7 +1573,11 @@ func RunVMIAndExpectLaunchWithDataVolume(vmi *v1.VirtualMachineInstance, dv *cdi
 	By("Waiting until the DataVolume is ready")
 	Eventually(ThisDV(dv), timeout).Should(HaveSucceeded())
 	By("Waiting until the VirtualMachineInstance will start")
-	WaitForSuccessfulVMIStartWithTimeout(obj, timeout)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	warningsIgnoreList := []string{"didn't find PVC associated with DataVolume"}
```

Comment:
ah, i don't think this will work because the warning is a strict match of this string, and not just checking if the warning contains the substring. 

Maybe the utils.go funciton that's comparing warnings should use `strings.Contains()` instead of `==` in the shouldIgnoreWarning() function

Reply:
[Done](https://github.com/kubevirt/kubevirt/pull/6407/commits/dcda5807c2df787fc2a3ed23f47fb2da1c056484)